### ISSUE TITLE
misc fixes for test email page

### ISSUE THREAD

**Issue by binary1230** [magfest/ubersystem#1408]
for /emails/test_email

1) make it CC the FROM address so people know that emails are being sent on their behalf
2) make it be accessible only by "dev" or "super-admin" user roles (Which don't exist yet)

**Comment by earl7399** [comment]
Is it currently only available to actual logged-in folks, and does its use get logged in the FODC?  B/C while it isn't technically a DC, it should REALLY be logged IMO

**Comment by EliAndrewC** [comment]
Good point.  We should make sure to include the ``model`` parameter (which doesn't have to be a model instance and can just be the string ``'n/a'`` which will cause this email to be recorded in the ``email`` table.

**Comment by binary1230** [comment]
remove value="submit" from HTML and it will not require **params to be passed in via cherrypy
